### PR TITLE
Add Base Web to UI Components

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

**Comment by gergelyke** [comment]
@alexkrolick is there anything missing for this PR to be landed?

File: content/community/tools-ui-components.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ permalink: community/ui-components.html
 ## Free Components {#free-components}
 * **[Amaze UI React](https://github.com/amazeui/amazeui-react) (in Chinese):** [Amaze UI](https://github.com/allmobilize/amazeui) components built with React.
 * **[Ant Design of React](https://github.com/ant-design/ant-design)** An enterprise-class UI design language and React-based implementation.
+* **[Base Web](http://baseweb.design):** Base Web is a foundation for initiating, evolving, and unifying web products.
```
Comment (lex111) [inline content/community/tools-ui-components.md]:
```suggestion
* **[Base Web](http://baseweb.design):** A foundation for initiating, evolving, and unifying web products.
```

**Comment by lex111** [comment]
@gergelyke thank you!

**Comment by gergelyke** [comment]
thanks for merging 🤗

**Pull request merged**